designsafe/settings/common_settings.py

```diff
@@ -569,6 +569,12 @@ def guid_filter(record):
 TAPIS_ADMIN_JWT = os.environ.get('TAPIS_ADMIN_JWT')
 TAPIS_TG458981_JWT = os.environ.get('TAPIS_TG458981_JWT')
 
+# Core Portal Configuration
+CORE_PORTAL_API_BASE = os.environ.get("CORE_PORTAL_API_BASE", "https://cep.tacc.utexas.edu/api/system-monitor")
+
+# Tapis API Configuration
+TAPIS_API_STATUS = os.environ.get("TAP_API_STATUS", "https://tap.tacc.utexas.edu/status/")
```

Comment:
```suggestion
TAP_API_STATUS = os.environ.get("TAP_API_STATUS", "https://tap.tacc.utexas.edu/status/")
```

Reply:
Sorry for the confusion, but TAP is the "TACC Analysis Portal" and Tapis is the name of the API we use for our HPCs.